### PR TITLE
Patch FS cleanup

### PR DESCRIPTION
**Pull request by Guillem96**

### CODE REVIEW

**Comment by Guillem96** [comment]
A quick cleanup of patch FS code. Now it is shorter, easier to understand and quicker (skips some iterations on foreach loop)

File: src/firmware.c [hunk 1]
```diff
@@ -224,54 +276,13 @@ void patch(pk11_offs *pk11, pkg2_hdr_t *pkg2, link_t *kips) {
...
-                }
-            }
-            
-            free(ki->kip1);
-            ki->size = newSize;
-            ki->kip1 = moddedKip;
-        }
+    pkg2_kip1_info_t* FS_module = find_by_tid(kips, 0x0100000000000000);
+    if(FS_module == NULL) {
+        error("     Could't find FS Module.\n");
```
[the remark this answers is not in the document]
Comment (Guillem96) [reply src/firmware.c]: Miss click with tab :)

**Comment by neonsea** [comment]
The line I mentioned still isn't fixed

**Comment by Guillem96** [comment]
@neonsea Take a look at my fork (https://github.com/Guillem96/ReiNX/blob/master/src/firmware.c). The indent is correct there.

**Comment by neonsea** [comment]
I'm talking about the error calls.

**Comment by neonsea** [comment]
Typo is still there... "Could't"

**Comment by Guillem96** [comment]
Closed because I was blind, so i did not see the spelling mistake after 3 commits.
New pull request is open with the correct FS patch clean up code

**Pull request closed without merging**